src/main/java/com/crowdin/client/ai/AIApi.java

```diff
@@ -0,0 +1,159 @@
+     * </ul>
+     */
+    public ResponseObject<FineTuningJob> createFineTuningJob(final Long userId, final Long aiPromptId, final FineTuningJobRequest request) {
+        String url = getFineTuningPath(userId, "ai/prompts/" + aiPromptId + "/fine-tuning/jobs");
+        FineTuningJobResponseObject responseObject = this.httpClient.post(url, request, new HttpRequestConfig(), FineTuningJobResponseObject.class);
+        return ResponseObject.of(responseObject.getData());
+    }
+
+    /**
+     * @param userId user identifier
+     * @param aiPromptId AI prompt identifier
+     * @param jobIdentifier AI prompt fine-tuning dataset generation identifier, consists of 36 characters
+     * @return AI Prompt Fine-Tuning Dataset Job Status
+     * @see <ul>
+     *     <li><a href="https://support.crowdin.com/developer/api/v2/#tag/AI/operation/api.users.ai.prompts.fine-tuning.jobs.get" target="_blank"><b>API Documentation</b></a></li>
+     *     <li><a href="https://support.crowdin.com/developer/enterprise/api/v2/#tag/AI/operation/api.ai.prompts.fine-tuning.jobs.get" target="_blank"><b>Enterprise API Documentation</b></li>
+     * </ul>
+     */
+    public ResponseObject<FineTuningJob> getFineTuningJobStatus(final Long userId, final Long aiPromptId, final String jobIdentifier) {
+        String url = getFineTuningPath(userId, "ai/prompts/" +  aiPromptId + "/fine-tuning/jobs/" + jobIdentifier);
+        FineTuningJobResponseObject responseObject = this.httpClient.get(url, new HttpRequestConfig(), FineTuningJobResponseObject.class);
+        return ResponseObject.of(responseObject.getData());
+    }
+
+    /**
+     * @param userId user identifier
+     * @param aiPromptId AI prompt identifier
+     * @param jobIdentifier AI prompt fine-tuning dataset generation identifier, consists of 36 characters
+     * @return AI Prompt Fine-Tuning Dataset Download URL
+     * @see <ul>
+     *     <li><a href="https://support.crowdin.com/developer/api/v2/#tag/AI/operation/api.users.ai.prompts.fine-tuning.datasets.download.get" target="_blank"><b>API Documentation</b></a></li>
+     *     <li><a href="https://support.crowdin.com/developer/enterprise/api/v2/#tag/AI/operation/api.ai.prompts.fine-tuning.datasets.download.get" target="_blank"><b>Enterprise API Documentation</b></li>
+     * </ul>
+     */
+    public ResponseObject<FineTuningDatasetDownload> downloadFineTuningDataset(final Long userId, final long aiPromptId, final String jobIdentifier) {
+        String url = getFineTuningPath(userId, "ai/prompts/" + aiPromptId + "/fine-tuning/datasets/" + jobIdentifier + "/download");
+        FineTuningDatasetDownloadResponse response = this.httpClient.get(url, new HttpRequestConfig(), FineTuningDatasetDownloadResponse.class);
+        return ResponseObject.of(response.getData());
+    }
+
+
+    private String getFineTuningPath(Long userId, String path) {
```

Comment:
This method could probably be reused later for other AI methods, so I would suggest using a generic name here, e.g. `getPathPrefix`.